pkg/login/social/socialimpl/service.go

```diff
@@ -72,27 +72,27 @@ func ProvideService(cfg *setting.Cfg,
 
 			settingsKVs := convertIniSectionToMap(sec)
 
-			info, err := connectors.CreateOAuthInfoFromKeyValues(settingsKVs)
+			conn, err := createOAuthConnector(name, settingsKVs, cfg, ssoSettings, features, cache)
```

Comment:
The `createOAuthConnector` should be after the "grafananet" check (https://github.com/grafana/grafana/pull/79459/files#diff-7e23901642ddb0d7b2664c23827498304bbe4e446a23e5a848e4020e096567d3R92)